docs/content/posts/2023-08-03-connector-updates.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "DynamoDB, MongoDB and OpenSearch externalized connectors"
+date: "2023-08-03T12:00:00Z"
+authors:
+- elphastori: 
+  name: "Elphas Toringepi"
+  twitter: "elphastori"
+aliases:
+- /news/2023/08/03/externalized-connectors.html
+---
+
+## New connectors
+
+We're excited to announce that Apache Flink now includes 3 new connectors for [DynamoDB](https://aws.amazon.com/dynamodb), [MongoDB](https://www.mongodb.com/) and [OpenSearch](https://opensearch.org/)! The connectors are available for both the [DataStream](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/overview/) and [Table/SQL](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/overview/) APIs.  
+
+- **[DynamoDB sink](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/dynamodb/)** - The DynamoDB sink writes to [Amazon DynamoDB](https://aws.amazon.com/dynamodb) using the [AWS v2 SDK for](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html) Java.
+- **[MongoDB connector](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/mongodb/)** - Flink now provides a [MongoDB](https://www.mongodb.com/) connector for reading and writing data from and to MongoDB collections with at-least-once guarantees.
+- **[OpenSearch sink](https://github.com/apache/flink-connector-opensearch/blob/main/docs/content/docs/connectors/datastream/opensearch.md)** - This connector provides a sink that can request document actions to an [OpenSearch](https://opensearch.org/) Index.
+
+|Connector|Date Released|Supported Flink Versions|
+|---|---|---|
+|DynamoDB sink|12/2/2022|1.15+|
+|MongoDB connector|3/31/2023|1.16+|
+|OpenSearch sink|12/21/2022|1.16+|
+
+## Externalized connectors
+
+In order to improve the overall connector ecosystem, we introduced the new connectors in separate repositories and moved existing connectors out of [Flink’s main repository](https://github.com/apache/flink).  
+
+This decouples the connectors' release cycle from Flink’s release cycle, resulting in faster releases of connectors. It also allows the addition of newer connector features to older Flink versions and facilitates more contributions to connectors, among numerous other benefits  
```

Comment:
```suggestion
The community have externalized connectors from [Flink’s main repository](https://github.com/apache/flink). This was driven to realise the following benefits:
- **Faster releases of connectors:** New features can be added more quickly, bugs can be fixed immediately, and we can have faster security patches in case of direct or indirect (through dependencies) security flaws.
- **Adding newer connector features to older Flink versions:** By having stable connector APIs, the same connector artifact may be used with different Flink versions. Thus, new features can also immediately be used with older Flink versions. 
- **More activity and contributions around connectors:** By easing the contribution and development process around connectors, we will see faster development and also more connectors.
- **Documentation:** Standardized documentation and user experience for the connectors, regardless of where they are maintained. 
- **A faster Flink CI:** By not needing to build and test connectors, the Flink CI pipeline will be faster and Flink developers will experience fewer build stabilities (which mostly come from connectors). That should speed up Flink development.
```